src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black,
+    Blue,
+    Green,
+    Grey,
+    Purpur,
+    White,
+    Yellow
```

Comment:
Don’t forget how to name constants according to style guide (the same with naming of enum values) (Should be capital letters)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names